workflows/sra_download.nf

```diff
@@ -93,6 +94,7 @@ workflow SRA_DOWNLOAD {
             .map { meta, reads -> if (!meta.fastq_1) "${meta.id.split('_')[0..-2].join('_')}" }
             .unique()
             .collectFile(name: no_ids_file, sort: true, newLine: true)
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/nf-core/viralrecon/pull/173/commits/dbb36937096edf588747e2ac715169afaf5b2e78